core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -185,18 +216,60 @@ public PlayerInformation getPlayerInformation() {
     public void attach(PlayerView playerView) {
         videoContainer = VideoContainer.with(playerView.getContainerView());
         playerView.simplePlayerView().setPlayer(exoPlayer);
-        // TODO : Set SubtitleView on the facade.
     }
 
     @Override
-    public void selectAudioTrack(int audioTrackIndex) {
-        //TODO: Select the audio track on the facade.
+    public void selectAudioTrack(PlayerAudioTrack audioTrack) {
+        MappingTrackSelector.MappedTrackInfo trackInfo = trackSelector.getCurrentMappedTrackInfo();
+        TrackGroupArray trackGroups = trackInfo.getTrackGroups(TRACK_TYPE_AUDIO);
+        FixedTrackSelection.Factory factory = new FixedTrackSelection.Factory();
+
+        MappingTrackSelector.SelectionOverride selectionOverride = new MappingTrackSelector.SelectionOverride(
+                factory,
+                audioTrack.groupIndex(),
+                audioTrack.trackIndex()
```

Comment:
We were originally going to use just an id for selecting a track but it actually means two iterations through a double for loop, once for retrieving the list and again for selecting. Even then we would need to potentially throw if we do not find anything, this seemed like the safest course of action.